### PR TITLE
update sajc

### PR DESCRIPTION
**Pull request by huber-nicolas**

### CODE REVIEW

File: file-formats/sajc/type/zif_aff_sajc_v1.intf.abap [hunk 1]
```diff
@@ -17,11 +17,15 @@ INTERFACE zif_aff_sajc_v1
...
+      abap_language_version TYPE zif_aff_types_v1=>ty_abap_language_version_cloud,
     END OF ty_header.
 
+  "! <p class="shorttext">Report Name</p>
+  "! Name of the report
+  TYPES ty_report_name TYPE c LENGTH 40.
+
   TYPES:
-    "! <p class="shorttext">Application Job Catalog Entry</p>
+    "! <p class="shorttext">General Information</p>
```
Comment (wurzka) [inline file-formats/sajc/type/zif_aff_sajc_v1.intf.abap:28]: I think we talked about this before a while: We do not want object typer owner to have "General Information" in their AFF type/schema. When they want to have it as section title in their editor, they have to add it to their configuration. So I would update this here.

**Review by wurzka — CHANGES_REQUESTED** [review]
See my comment

File: file-formats/sajc/type/zif_aff_sajc_v1.intf.abap [hunk 2]
```diff
@@ -34,6 +38,9 @@ INTERFACE zif_aff_sajc_v1
       "! Name of the class which contains the execute-method to run within the job
       "! $required
       class_name            TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Executed Report</p>
+      "! Name of the report which is executed within the job
+      report_name           TYPE ty_report_name,
```
Comment (schneidermic0) [inline file-formats/sajc/type/zif_aff_sajc_v1.intf.abap:43]: I just came across "report". Can we change this to "program"? As far as I can see in SAPterm. "report" is the obsolete term for "executable program".
```suggestion
      "! <p class="shorttext">Executed Program</p>
      "! Name of the program which is executed within the job
      program_name           TYPE ty_report_name,
```

**Review by schneidermic0 — CHANGES_REQUESTED** [review]
Sorry, @huber-nicolas. I just found some more issues :(

File: file-formats/sajc/README.md [hunk 3]
```diff
@@ -2,4 +2,4 @@
 
 File | Cardinality | Definition | Schema | Example
 :--- | :---  | :--- | :--- | :---
-`<name>.sajc.json` | 1 | [`zif_aff_sajc_v1.intf.abap`](./type/zif_aff_sajc_v1.intf.abap) | [`sajc-v1.json`](./sajc-v1.json) | [`z_aff_example_sajc.sajc.json`](./examples/z_aff_example_sajc.sajc.json)
+`<name>.sajc.json` | 1 | [`zif_aff_sajc_v1.intf.abap`](./type/zif_aff_sajc_v1.intf.abap) |[`sajc-v1.json`](./sajc-v1.json) | [`z_aff_example_sajc.sajc.json`](./examples/z_aff_example_sajc.sajc.json)
```
[the remark this answers is not in the document]
Comment (huber-nicolas) [reply file-formats/sajc/README.md:5]: yes this can be reverted

**Review by wurzka — APPROVED** [review]

**Review by schneidermic0 — APPROVED** [review]
Looks good to me

**Pull request merged**